src/en/developer-event-cycle.md

```diff
@@ -38,7 +38,7 @@ Reference section of the Juju documentation.
 
 When deploying a charm, there is a guaranteed set lifecycle events that every
 charm will run - this is the basic series of hooks executed in the initial
-deployment cycle.  
+deployment cycle.
```

Comment:
Actually in this case the two spaces at the end of a line have a meaning to start a new paragraph.  I have had some problems with numbered lists right after text paragraphs so I use the two spaces on purpose as I did here.